metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -322,6 +335,18 @@ def __init__(self, config: SQLCommonConfig, ctx: PipelineContext, platform: str)
                 cached_domains=[k for k in self.config.domain], graph=self.ctx.graph
             )
 
+        self.views_processed: Set[str] = set()
+        self.schema_resolver: SchemaResolver = SchemaResolver(
+            platform=self.platform,
+            platform_instance=self.config.platform_instance,
+            env=self.config.env,
+        )
+        self._view_definition_cache: MutableMapping[str, str]
+        if self.config.use_file_backed_cache:
+            self._view_definition_cache = FileBackedDict[str]()
+        else:
+            self._view_definition_cache = {}
+
```

Comment:
moved `use_file_backed_cache` related logic from teraform source here.

Reply:
Now that we fixed the file backed dict to support windows, imo we can probably drop this config flag

Reply:
Can you point me to the PR that fixes this ?